drv/mr_timer_hf/mr_timer_hf.c

```diff
@@ -116,6 +117,7 @@ void mr_timer_hf_init(timer_hf_t timer) {
     _devs[timer].p->PRESCALER   = 4;  // Run TIMER at 1MHz
     _devs[timer].p->BITMODE     = (TIMER_BITMODE_BITMODE_32Bit << TIMER_BITMODE_BITMODE_Pos);
     _devs[timer].p->INTENSET    = (1 << (TIMER_INTENSET_COMPARE0_Pos + _devs[timer].cc_num));
+    NVIC_SetPriority(_devs[timer].irq, TIMER_IRQ_PRIORITY);
```

Comment:
This is only needed in the gateway, so it should not be applied for the nodes.

Reply:
We will not use a feature flag to simplify and have everything just working by default.